app/controllers/info_controller.rb

```diff
@@ -19,20 +19,12 @@ def show
     begin
       @content = content_store.content_item("/#{@slug}").to_h
       @needs = @content["links"]["meets_user_needs"]
-    rescue GdsApi::ContentStore::ItemNotFound
-      if metadata
-        # If the content store does not have this content, fall back
-        # to the Metadata API, which may be able to fetch the content
-        # from the content-api
-        @content = metadata.fetch("artefact")
-        valid_needs = metadata.fetch("needs").select do |need|
-          need["status"]["description"] == "valid"
-        end
-        @needs = valid_needs.map { |need| { "details" => need } }
-      end
+    rescue GdsApi::ContentStore::ItemNotFound => e
+      # Suppress the error - this is handled by @content being nil at the end of the action
+      Rails.logger.error(e)
```

Comment:
This probably doesn't want logging, as someone going to a info page for some non-existent content is not an error. Also, as a 404 is returned, with the current GOV.UK setup, this information will be available from the NGinx logs.